org-gtd-refile.el

```diff
@@ -34,6 +34,11 @@
 
 ;;;; Customization
 
+(defgroup org-gtd-refile nil
+  "Customize how tasks within the GTD system are refiled."
+  :group 'org-gtd
+  :package-version '(org-gtd . "3.0.0"))
```

Comment:
Maybe you want to adjust the `:package-version` value here?

Reply:
There's no reason for this to exist at the moment.
Maybe you intended to add the defcustom statement below to it, but that would be a business logic change, because this change impacts how the user interacts with the package.

If you want to suggest this change, it needs to be in a separate PR so we can talk about it in details, it may trigger further changes.

Reply:
I see. My rationale was that I thought we would all the customizable settings to be visible under the `org-gtd` group in the Customize interface. Is there a way I can remove this change from the PR. Undo the change and push again?

Reply:
Raised as issue #158.